crates/madara/src/commands/bootstrapper.rs

```diff
@@ -13,3 +19,31 @@ pub fn build_image(shell: &Shell) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub fn process_params(global_config: &Config) -> anyhow::Result<()> {
```

Comment:
Could be useful to add a validate() func for `Config` here

Reply:
Validation function is applied to custom configuration file (the one that the user can generate). If no file is provided, the template will be used and that must always be valid and working.
Besides, a better and more modularized approach was used